src/components/HookCounter/HookCounterTwo.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useState } from "react";
+function HookCounterTwo(){
+    const initialCount=0
+    const [count, setCount]=useState(initialCount)
+
+    const incrementFive = () => {
+        for(let i=0;i<5; i++) {
+            setCount(prevCount => prevCount + 1)
```

Comment:
any reason using loop and incrementing by 1, you can directly increment 5 in a single go right

Reply:
Yes, but just to practice looping I tried it.